src/const.ts

```diff
@@ -316,6 +317,11 @@ interface ConstantFeatureFlags {
    * @description Enable widget expand button.
    * */
   enableWidgetExpandButton: boolean;
+  /**
+   * @public
+   * @description Enable message grouping by timestamp.
+   * */
+  enableMessageGrouping: boolean;
```

Comment:
need this for snapshot test to produce same results that are not based on current time.